src/pages/installing-spark/angular.mdx

```diff
@@ -3,23 +3,20 @@
   order: 4
 ---
 import styleCheck from '../../images/installing-spark/angular/style-check.png';
-import spinnergif from '../../images/installing-spark/angular/angular-spinner.gif';
+import spinnergif from '../../images/installing-spark/html/html-spinner.gif';
```

Comment:
We're using the same image for all docs. It looks the same anyways:
![html-spinner](https://user-images.githubusercontent.com/4342363/119891410-eeadab00-bf06-11eb-8861-bf222be6ac0b.gif)